src/components/modules/toolbar/index.ts

```diff
@@ -167,7 +167,6 @@ export default class Toolbar extends Module<ToolbarNodes> {
       opened: this.toolboxInstance.opened,
       close: (): void => {
         this.toolboxInstance.close();
-        this.Editor.Caret.setToBlock(this.Editor.BlockManager.currentBlock);
```

Comment:
@neSpecc 
didn't get why this line is needed, so removed it. If it is needed, it can be executed on condition or moved to some other place

Reply:
Try this:

1) Place the caret to the empty block
2) Press Tab to open the Toolbox
3) Press Esc to close the Toolbox

The Caret should return to the empty block. In case of removing this line that wouldn't happenn.

Reply:
Ok, moved this line to function called on esc press